src/main/webapp/nav.html

```diff
@@ -1,8 +1,9 @@
-
 <link rel="stylesheet" href="styles/general.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Is the font-awesome import necessary here?  It is used for icons.

Reply:
i dont think so ill remove it